hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -291,6 +296,12 @@ public static class Config implements Serializable {
     @Parameter(names = {"--validate-bloom-filters"}, description = "Validate bloom filters of base files", required = false)
     public boolean validateBloomFilters = false;
 
+    @Parameter(names = {"--view-storage-type-fs-listing"}, description = "View storage type to use for File System based listing", required = false)
+    public String viewStorageTypeForFSListing = FileSystemViewStorageType.MEMORY.name();
+
+    @Parameter(names = {"--view-storage-type-mdt"}, description = "View storage type to use for metadata based listing", required = false)
+    public String viewStorageTypeForMetadata = FileSystemViewStorageType.MEMORY.name();
```

Comment:
Is there a reason to support different view storage types for FS-based and MDT-based file listing?

Reply:
yes. for FS based listing, we wanted to keep it plain simple. what in case spillable map had some bugs and so we don't want to compare FS w/ spillable w/ MDT w/ FS based.